apps/notifications/package.json

```diff
@@ -28,8 +28,11 @@
 		"@automattic/calypso-color-schemes": "workspace:^",
 		"@automattic/calypso-polyfills": "workspace:^",
 		"@automattic/calypso-router": "workspace:^",
+		"@automattic/components": "workspace:^",
 		"@automattic/i18n-utils": "workspace:^",
+		"@automattic/viewport": "workspace:^",
 		"@automattic/webpack-extensive-lodash-replacement-plugin": "workspace:^",
+		"@wordpress/data": "^10.0.0",
```

Comment:
Apparently this is required by the components package, but the components package only lists it as a peerDependency, so we need to ensure its installed here.